src/lofreq/lofreq_uniq.c

```diff
@@ -446,6 +446,7 @@ main_uniq(int argc, char *argv[])
      static int use_det_lim = 0;
      static int use_orphan = 0;
      static int output_all = 0;
+     static int is_somatic = 0;
```

Comment:
Here, in line 482, and in 647, some code was added in your python3 branch with respect to your master.